tests/e2e/api/src/models/coupons/shared/update-params.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Coupon properties that can be updated
+ */
+export type CouponUpdateParams = 'code' | 'amount' | 'description' | 'dateExpires' | 'individualUse'
+  | 'usageCount' | 'productIds' | 'excludedProductIds' | 'usageLimit' | 'usageLimitPerUser' | 'limitUsageToXItems'
+  | 'freeShipping' | 'productCategories' | 'excludedProductCategories' | 'excludeSaleItems' | 'minimumAmount'
+  | 'maximumAmount' | 'emailRestrictions';
+
+/**
+ * The discount type of the coupon
+ */
+export type CouponDiscountTypeUpdateParams = 'percent' | 'fixed_cart' | 'fixed_product';
```

Comment:
The `UpdateParams` type in coupons.ts is a list of transformed field names. It should include `discountType` instead of the coupon types.

Reply:
Good call out--I've fixed this up.